### ISSUE TITLE
Collaborations bug

### ISSUE THREAD

**Issue by negarrahmati** [StanfordHCI/flash-teams#244]
When we finished two tasks that had collaborations, the remaining tasks got shifted twice.

**Comment by ato1120** [comment]
*Not necessarily having a "collaboration" just that they overlap and both finish early?

**Comment by negarrahmati** [comment]
Yep. Also, even if one of the concurrent tasks finishes early, the
remaining events shouldn't get moved to the left.

Negar
